app/scripts/modules/core/styleguide/public/styleguide.html

```diff
@@ -0,0 +1,1151 @@
+        <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,900" rel="stylesheet">
+        <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+    </head>
+
+    <body>
```

Comment:
Since this runs embedded in Deck now, can probably omit the `<html>`, `<head>`, and `<body>` tags, no?

Also, the fonts (Source Sans Pro, Font Awesome) should come along for free, so don't need to be linked in here.